source/developer-guides/contributions.rst

```diff
@@ -0,0 +1,30 @@
+.. _contribution_guidelines:
+
+Contribution Guidelines
+-----------------------
+
+As an open-source project, we welcome and encourage the community to
+submit patches directly to Slim bootloader.  
+
+Licensing
+*********
+
+Slim bootloader project uses a BSD-2 clause license, as found in the
+`LICENSE <https://github.com/slimbootloader/slimbootloader/blob/master/LICENSE>`__
+in the project's GitHub repo.
+
+It is important that the contributor fully understands the licensing terms and conditions 
+and agrees to them. 
+
+Coding Style
+************
+
+Slim bootloader project follows the `EDKII coding guidelines <https://github.com/tianocore/tianocore.github.io/wiki/Code-Style-C>`__
+ 
+
+.. _Contribution workflow:
+
+Contribution Workflow
+*********************
+
+Slim bootloader uses the `GitHub workflow <https://guides.github.com/introduction/flow/>`__
```

Comment:
The URL below provides more hands-on examples.    Might be better to use this instead.
https://guides.github.com/introduction/git-handbook/